ERESOLVE unable to resolve dependency tree

Is react-plaid-link not compatible with react17? 

```
npm install react-plaid-link       
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: frontend@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from react-plaid-link@3.1.0
npm ERR! node_modules/react-plaid-link
npm ERR!   react-plaid-link@"*" from the root project
```

I'm new to using react so please excuse my ignorance.

--- a/package.json
+++ b/package.json
@@ -46,8 +46,8 @@
     "react-script-hook": "1.1.0"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react": "^16.8.0 || ^17.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.7.0",

